src/main/java/org/jabref/gui/preferences/GeneralTabViewModel.java

```diff
@@ -136,7 +132,7 @@ public void storeSettings() {
 
         preferencesService.storeGeneralPreferences(new GeneralPreferences(
                 selectedEncodingProperty.getValue(),
-                selectedBiblatexModeProperty.getValue(),
+                ((JabRefPreferences) preferencesService).getBoolean(JabRefPreferences.BIBLATEX_DEFAULT_MODE) ? BibDatabaseMode.BIBLATEX : BibDatabaseMode.BIBTEX,
```

Comment:
Where does the user have the possibility to change that value (`JabRefPreferences.BIBLATEX_DEFAULT_MODE`)? Is it more a constant now?

Reply:
You mean the variable itself or the value it point to in JabRefPreferences.prefs? If you mean the variable itself, the access modifier for it is final, so I think it is a constant. If you mean the value it point to in JabRefPreferences.prefs, I think GeneralTabViewModel.storeSettings() does something to change it.

Reply:
Hi, thank you for your interest in preferences refactoring and simplifying. However, there are some architectural decisions we probably failed yet to communicate enough publicly.

Please do not use direct calls to JabRefPreferences::get etc. anymore. We are working in abstracting access to the preferences to modularize JabRef in the long run.
You can use preferences objects instead which can be received by calls to a PreferencesService. (getGeneralPreferences or similar)

Besides that, I agree with @Siedlerchr , there are some places and case scenarios, people will need to rely on bibtex mode instead of biblatex as default. So we shouldn't kill a feature in JabRef, without a strong necessity.

Thank you anyways. Don't hesitate to ask us about the roadmap of JabRef, about ideas you have and want to realize etc. The JabRef developer chat in gitter is open for discussion. Free software is meant for developers to discuss things and to help each other.